dbt/adapters/impala/connections.py

```diff
@@ -60,14 +59,35 @@ class ImpalaCredentials(Credentials):
         'user':'username'
     }
 
+    @classmethod
+    def __pre_deserialize__(cls, data):
+        data = super().__pre_deserialize__(data)
+        if 'database' not in data:
+            data['database'] = None
+        return data
+
+    def __post_init__(self):
+        # impala classifies database and schema as the same thing
+        if (
```

Comment:
nitpick formatting. Add a linter to format this.

Reply:
Not sure what you mean here, looks like this is the formatting I get from my linter :)